.envrc

```diff
@@ -0,0 +1 @@
+use nix
```

Comment:
Since we're using a flake it's a good idea to add.

```
watch_file flake.nix
watch_file flake.lock
```
so that direnv reloads if those files change.